indexer/test/indexer.test.js

```diff
@@ -250,6 +294,9 @@ describe('retriever.indexer', () => {
       const railId = randomId()
       const req = new Request('https://host/proof-set-rail-created', {
         method: 'POST',
+        headers: {
+          [env.SECRET_HEADER_KEY]: env.SECRET_HEADER_VALUE,
+        },
```

Comment:
Is it time to extract a helper function so that we don't have to repeat this header configuration in every test, and we don't have to update every test if we introduce more headers in the future?

```js
async function invokeWebhook(endpointPath, body) {
  const req = new Request(`https://host${endpointPath}`, {
    method: 'POST',
    headers: {
      [env.SECRET_HEADER_KEY]: env.SECRET_HEADER_VALUE,
    },
    body: JSON.stringify(body)
  )
  return fetch(req)
}
```

Reply:
I don't see a need for refactors at this point